master/docs/examples/git_gerrit.cfg

```diff
@@ -184,7 +184,15 @@ def gerritSummaryCB(buildInfoList, results, status, arg):
                 })
 
 c['buildbotURL'] = 'http://buildbot.example.com/'
+
+# The 'GerritStatusPush' target is used to publish build results in Gerrit
+# To create message (with Code-Review) used function 'gerritReviewCB'
+# Notification about this message will be sent to to all (change owners,
+# reviewers, watchers and any user who has starred the change)
+# https://gerrit-documentation.storage.googleapis.com/Documentation/2.9/cmd-review.html#_options
+
 c['status'].append(status.GerritStatusPush(gerrit_url, gerrit_user,
+                                           notify='ALL',
```

Comment:
Official documentation said nothing about it.
Before 2.9 default behaviour was `ALL`
In 2.9 seems to no `NONE`: http://osdir.com/ml/repo-discuss/2014-07/msg00350.html
In current version of Gerrit I don't know default behaviour :)